targeteditor/src/main/scala/explore/targeteditor/TargetBody.scala

```diff
@@ -85,6 +85,15 @@ object TargetBody extends ModelOptics {
               )
             }
           ) _
+          val modifyName = undoSet[NonEmptyString](
+            SiderealTarget.name,
+            n =>
+              TargetsSetInput(
+                name = n.value.some,
+                ra = none,
+                dec = none
```

Comment:
I think these values default to `None` for cases like these where we don't want to change them.